ui/static/app/http-client.ts

```diff
@@ -12,14 +12,12 @@ export class HttpClient {
   constructor(
     private http: Http,
     private authService: AuthService
-  ) {
-    this.apiToken = this.authService.getToken();
-  }
+  ) { }
 
   getAuthorizationHeader(): Headers {
     let headers: Headers = new Headers();
     headers.append("Content-Type", "application/json");
-    headers.append("Authorization", `Token ${this.apiToken}`);
+    headers.append("Authorization", `Token ${this.authService.getToken()}`);
```

Comment:
No, its ok, look:
https://github.com/allegro/django-powerdns-dnssec/blob/develop/ui/static/app/auth/auth.service.ts#L43

Token is getting only one in login page. 
Earlier token was taken from localStorage only when in create object from class.
Now is always get from  localStorage.